app/components/forms/wizard/other-details-step.js

```diff
@@ -0,0 +1,160 @@
+  }),
+
+  socialLinks: computed('data.event.socialLinks.@each.isDeleted', function() {
+    return this.data.event.socialLinks.filterBy('isDeleted', false);
+  }),
+
+  isUserUnverified: computed('authManager.currentUser.isVerified', function() {
+    return !this.authManager.currentUser.isVerified;
+  }),
+  /**
+   * returns the validation rules for the social links.
+   */
+  socialLinksValidationRules: computed('socialLinks', function() {
+    let validationRules = {};
+    for (let i = 0; i < this.socialLinks.length; i++) {
+      validationRules = merge(validationRules, {
+        [this.socialLinks.get(i).identifier]: {
+          identifier : this.socialLinks.get(i).identifier,
+          optional   : true,
+          rules      : [
+            {
+              type   : 'regExp',
+              value  : protocolLessValidUrlPattern,
+              prompt : this.l10n.t('Please enter a valid url')
+            }
+          ]
+        }
+      });
+    }
+
+    return validationRules;
+  }),
+
+  showDraftButton: computed('data.event.state', function() {
+    return this.data.event.state !== 'published';
+  }),
+
+  hasPaidTickets: computed('data.event.tickets.@each.type', function() {
+    return this.data.event.tickets.toArray().filter(ticket => ticket.type === 'paid' || ticket.type === 'donation').length > 0;
+  }),
+
+  hasCodeOfConduct: computed('data.event.codeOfConduct', function() {
+    return !!this.data.event.codeOfConduct;
+  }),
+
+  didInsertElement() {
+    if (!this.isCreate && this.data.event.copyright && !this.data.event.copyright.content) {
+      this.set('data.event.copyright', this.store.createRecord('event-copyright'));
+    }
+  },
+
+  // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
+  getValidationRules() {
+    $.fn.form.settings.rules.checkMaxMinPrice = () => {
+      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+    };
+    $.fn.form.settings.rules.checkMaxMinOrder = () => {
+      return $('.ui.form').form('get value', 'ticket_min_order') <= $('.ui.form').form('get value', 'ticket_max_order');
+    };
+
+    const validationRules = {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        externalEventIdentifier: {
+          identifier : 'external_event_url',
+          optional   : true,
+          rules      : [
+            {
+              type   : 'regExp',
+              value  : protocolLessValidUrlPattern,
+              prompt : this.l10n.t('Please enter a valid url')
+            }
+          ]
+        }
+      }
+    };
+    // Merging the predetermined rules with the rules for social links.
+    validationRules.fields = merge(validationRules.fields, this.socialLinksValidationRules);
+    return validationRules;
+  },
+
+  actions: {  
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6089057)